2.1_first_module.ipynb

```diff
@@ -228,7 +323,11 @@
    "source": [
     "What's going on? The test accepts a `Passthrough` module, assigns values to the module's inputs, and checks its outputs. To set an input, we call `poke`. To check an output, we call `expect`. If we don't want to compare the output to an expected value (no assertion), we can `peek` the output instead.\n",
     "\n",
-    "If all `expect` statements are true, then our boilerplate code will return true (see `testResult`)."
+    "If all `expect` statements are true, then our boilerplate code will return pass.\n",
+    "\n",
+    ">Note that the `poke` and `expect` use chisel hardware literal notation. Both operations expect literals of the correct type.\n",
+    "If `poke`ing a `UInt()` you must supply a `UInt` literal (example: `c.io.in.poke(10.U)`, likewise if the input is a `Bool()` the `poke` would expect either `true.B` or `false.B`.\n",
```

Comment:
Might also be useful to provide a negative example for each, eg for UInt, "this is distinct from a Scala number, eg `10`, which will not work and type-error"